client/asset/zec/zec.go

```diff
@@ -185,6 +193,12 @@ func NewWallet(cfg *asset.WalletConfig, logger dex.Logger, net dex.Network) (ass
 		SingularWallet:           true,
 		UnlockSpends:             true,
 		FeeEstimator:             estimateFee,
+		ConnectFunc: func() error {
+			return connect(w)
+		},
+		AddrFunc: func() (btcutil.Address, error) {
+			return transparentAddress(w, addrParams, btcParams)
```

Comment:
I will look closer at this, but I think I love this capture of the rpcCaller vs the awkwardness we ensure with `estimateFee(ctx context.Context, node btc.RawRequester, ...)`.
Seems like in an other commit we could follow this pattern and eliminate a bunch of cruft like `btc.RawRequester` itself.